examples/utils/example-utils/src/migrator/sameContainerMigrator.ts

```diff
@@ -144,7 +160,7 @@ export class SameContainerMigrator
 				const detachedModel = await this.modelLoader.createDetached(acceptedVersion);
 				const migratedModel = detachedModel.model;
 
-				const exportedData = await migratable.exportData();
+				const exportedData = await this.pausedModel.exportData();
```

Comment:
This probably also gets safer when we can directly await the load here - otherwise I think this is implicitly expecting MigrationTool won't reach "readyForMigration" state without first awaiting the loadPausedContainerCallback (and thereby ensuring the pausedModel won't still be undefined here)